queries.sql

```diff
@@ -38,4 +47,13 @@ SELECT * FROM animals;
 UPDATE animals SET weight_kg= weight_kg * -1 WHERE weight_kg < 0;
 SELECT * FROM animals;
 
-COMMIT;
\ No newline at end of file
+COMMIT;
+
+-- queies
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT COUNT(escape_attempts) as count, neutered FROM animals GROUP BY neutered ORDER BY count DESC LIMIT 1;
```

Comment:
- [ ] I'd suggest you update the query for the question "Who escapes the most, neutered or not neutered animals?" in line `57` as the `escape_attempts` column already holds the number of escape attempts you can either use `AVG` or `MAX` aggregate function as using `COUNT` wouldn't provide a proper result.